### PR TITLE
Add utility to extract and represent emoji ordering information.

### PR DESCRIPTION
**Pull request by dougfelt**
This parses either a .csv file or emojiOrdering.txt file that contain
information about grouping and ordering of emoji in categories and
subcategories.  It defines a class that provides access to this
data.

### CODE REVIEW

**Comment by dougfelt** [comment]
ping

File: nototools/emoji_ordering.py [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+import argparse
+import codecs
+import collections
+from itertools import chain
+from os import path
+import re
+import sys
+
+def _generate_emoji_to_cat_tuple(category_odict):
```
Comment (jamesgk) [inline nototools/emoji_ordering.py]: Could/should this be a method of EmojiOrdering?

**Comment by jamesgk** [comment]
LGTM

File: nototools/emoji_ordering.py (hunk 1 above)
Comment (dougfelt) [inline nototools/emoji_ordering.py]: Could.  The line between what's in the class and what's not is a little fuzzy.  The data structure, for example, is built outside the class by the parsing code.  The conversion of the data structure to the inverse map seemed to me more like a utility function that could be used elsewhere (e.g. to verify the original data structure doesn't contain multiple mappings for the same sequence, which I currently don't do) so I left it out of the class.  I'll probably leave it as is.

**Pull request merged**